lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -18,29 +18,29 @@ module FlashHelper
       :warning   => :warning,
       :primary   => :primary
     }
-    def display_flash_messages(key_matching = {})
-      key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+    def display_flash_messages(**options)
```

Comment:
Could we have closable as a second argument instead? Maybe a keyword argument?

`key_matching` is a lookup, not an options hash, so it doesn't make sense to me to pass `:closable` as part of the existing hash.

I think the best implementation would be to use keyword arguments for both - something like:

```ruby
def display_flash_messages(closable: true, key_matching: {})
```

That would be a breaking change, but then again moving to foundation 6 is a breaking change, so I don't see that as a reason not to do it. Maybe we'd need to add some upgrade notes.

An alternative would be an options hash, which is essentially the same, but a little more verbose - something like:

```ruby
def display_flash_messages(options: {})
  closable = options.fetch(:closable, true)
  new_key_matching = options[:key_matching] || {}
  key_matching = DEFAULT_KEY_MATCHING.merge(new_key_matching)
```